### ISSUE TITLE
Media Watch Folder Notification Says "Incoming" But Never Imports

### ISSUE THREAD

**Issue by jacktheripper96** [fcpxhacks/fcpxhacks#3371]
<img width="409" alt="Screenshot 2024-08-26 at 9 07 18 AM" src="https://github.com/user-attachments/assets/30365b9c-3d7e-4670-80b1-911ac8f51dd7">

Not given the option to import files. Gets stuck

**Comment by latenitefilms** [comment]
To be honest, we haven't looked at the Watch Folder code in years... there's only a handful of people that actually use this functionality and they're all on very old versions of CommandPost. I'll try and reproduce the issue.

What version of macOS? What version of Final Cut Pro? What version of CommandPost?

**Comment by jacktheripper96** [comment]
Yeah, just thought it would be nice to have CP auto import files that I dump into a project folder. Saves a step. I have a buddy who found a similar app for premiere and it saves a bunch of time.  

FCP 10.8
CP 1.4.27
Sonoma 14.6.1

**Comment by latenitefilms** [comment]
I'll have a proper look tomorrow.

**Comment by jacktheripper96** [comment]
Any luck?

**Comment by latenitefilms** [comment]
It looks like it's working fine here:

<img width="390" alt="image" src="https://github.com/user-attachments/assets/6b422c78-acee-46fb-8498-78549ca873ab">

Using these settings:

<img width="1000" alt="image" src="https://github.com/user-attachments/assets/daa3c3d7-ac68-4cc2-b516-37922c78ea53">
